scipy/sparse/linalg/_dsolve/linsolve.py

```diff
@@ -222,7 +222,7 @@ def spsolve(A, b, permc_spec=None, use_umfpack=True):
             row_segs = []
             col_segs = []
             for j in range(b.shape[1]):
-                bj = b[:, j].toarray().ravel()
+                bj = b[:, [j]].toarray().ravel()
```

Comment:
Leave a TODO comment here to revert this change once sparrays support 1d indexing.

Reply:
I think in this case it makes no difference, since the array is first densified and then raveled.

Reply:
OK, added a note.